demo/demo-spring-boot-transport/demo-spring-boot-pojo-server/src/main/java/io/servicecomb/demo/pojo/server/HelloImpl.java

```diff
@@ -16,15 +16,28 @@
 
 package io.servicecomb.demo.pojo.server;
 
+import javax.ws.rs.core.MediaType;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
 import io.servicecomb.demo.helloworld.greeter.Hello;
+import io.servicecomb.provider.pojo.RpcSchema;
 
+@RpcSchema(schemaId = "helloworld.Greeter")
```

Comment:
why old one no need this annotations?

Reply:
In old edition, the schema of HelloImpl is configured in a xml file. But the schema of TestImpl is configured by annotation. After discussion with Liu Bao, we think that two kind of configuration existing in one demo is confusing, so I transform it into annotation config.